pkgs/development/python-modules/symspellpy/default.nix

```diff
@@ -31,6 +32,13 @@ buildPythonPackage rec {
       url = "https://patch-diff.githubusercontent.com/raw/mammothb/symspellpy/pull/150.patch";
       hash = "sha256-mdUJMrcPv5zczIRP+8u5vicz2IE1AUN3YP0+zg3jqZg=";
     })
+  ] ++ lib.optionals (pythonAtLeast "3.12") [
```

Comment:
If the patch is safe to apply to python3.11 and below, please do it for maintainability.

Reply:
duplicate